src/components/Global/TotalValue.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+import { CountUp } from 'countup.js'
```

Comment:
There is a [react-countup](https://github.com/glennreyes/react-countup) component. What do you think of using it ? Does it fills your needs ?